test/backend/datastores.livy.spec.js

```diff
@@ -76,7 +76,8 @@ df.registerTempTable("ALCOHOL_CONSUMPTION_BY_COUNTRY_2010")
             'ALCOHOL_CONSUMPTION_BY_COUNTRY_2010';
 
         return query(`SELECT * FROM ${tableName}\nLIMIT 5`, connection).then(results => {
-            assert.deepEqual(results.rows, [
+            const roundedToSinglePrecision = results.rows.map(r => [r[0], Number(r[1].toPrecision(6))]);
```

Comment:
It looks like when the table was created in Spark, the source CSV file was imported as single-precision numbers. But when we query Spark/Livy, the results are reported as double-precision numbers.

@chriddyp Have you come across this issue with any of the other connectors?